data/README.md

```diff
@@ -0,0 +1,163 @@
+# Attribution
+
+The scripts provided here wrap around the code related to `CTSM/tools/site_and_regional/subset_data` for single point data creation, available in [the CTSM repository](https://github.com/ESCOMP/CTSM/tree/ctsm5.1.dev112/tools/site_and_regional). A tutorial is available [here](https://github.com/NCAR/CTSM-Tutorial-2022/blob/main/notebooks/Day2a_GenericSinglePoint.ipynb). Please ensure to attribute this work if you use this code.
+
+_Kaveh Karimi & Lasse Keetz, 31-10-2021._
+
+# Instructions to create input data for a custom site
+
+The scripts to create new input data with this `ctsm-api` version require the following global datasets and file structures:
+
+- `<cesm-data-root>/atm/datm7/atm_forcing.datm7.GSWP3.0.5d.v1.c170516`
+- `<cesm-data-root>/share/domains/domain.lnd.fv0.9x1.25_gx1v7.151020.nc`
+- `<cesm-data-root>/lnd/clm2/surfdata_map/release-clm5.0.18/surfdata_0.9x1.25_hist_16pfts_Irrig_CMIP6_simyr2000_c190214.nc`
+- `<cesm-data-root>/lnd/clm2/surfdata_map/release-clm5.0.18/surfdata_0.9x1.25_hist_78pfts_CMIP6_simyr2000_c190214.nc`
+
+The datasets are available at https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/.
+
+If you want to use different versions of these data (WARNING! Might break the model, only if you know what you are doing!), make sure they exist in the given paths and adapt the names of the files in:
+
+- `<ctsm-api-root>/data/create_data.py` -> Adapt name of atmospheric forcing
+- `<ctsm-root>/tools/site_and_regional/default_data.cfg` -> Adapt other file names
+
+This document provides a working example tested on a remote server ([SAGA](https://documentation.sigma2.no/hpc_machines/saga.html])) with Anaconda. The time it takes to create the data is dependent on the machine and the amount of cores you use; on Saga, creating data for a single site requires approximately 1.5 hours.

```

Comment:
There's an extra bracket at the end of the url.

Reply:
This is on purpose! Its "(SAGA)" as a hyperlink in parentheses :)

Reply:
Whoops, nice catch!! Fixed now.